workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-site-initializer/site-initializer/layout-page-templates/master-pages/marketplace-authorize/master-page.json

```
@@ -0,0 +1,3 @@
+{
+	"name": "Marketplace Authorize"
```

Rename the folder to marketplace-blank

```suggestion
	"name": "Marketplace Blank"
```